filemage/tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py38
+envlist =
+    py38
+
+[testenv]
+ensure_default_envdir = true
+envdir =
+    py38: {toxworkdir}/py38
+description =
+    py38: e2e ready
```

Comment:
> Azure CI pipeline is failing at E2E tests, is there a way to skip these? I did not provide any docker compose environments or E2E tests. If there is some need for this I can easily build out the docker env, but I don;t understand how these `E2E` tests are intended to be built / run in your CI pipeline. Some advice would be helpful here

I think that removing these lines should disable e2e tests.

```suggestion
```

Reply:
resolved in 9cfba8d6052b58fb9a08c256eedc64c59ae918b1